content/browser/rewards_data/rewards_data_remover_impl.h

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef BRAVE_CONTENT_BROWSER_REWARDS_DATA_REWARDS_DATA_REMOVER_IMPL_H_
+#define BRAVE_CONTENT_BROWSER_REWARDS_DATA_REWARDS_DATA_REMOVER_IMPL_H_
+
+#include <stdint.h>
+
+#include <set>
+
+#include "base/containers/queue.h"
+#include "base/gtest_prod_util.h"
+#include "base/macros.h"
+#include "base/memory/weak_ptr.h"
+#include "base/observer_list.h"
+#include "base/supports_user_data.h"
+#include "brave/content/public/browser/rewards_context.h"
+#include "brave/content/public/browser/rewards_data_remover.h"
+#include "content/common/content_export.h"
+
+namespace content {
+
+class BrowserContext;
+
+class CONTENT_EXPORT RewardsDataRemoverImpl
```

Comment:
this should not have `CONTENT_EXPORT`. That is a macro for the content module

Reply:
This file was removed